remoteappmanager/static/js/home/components/application_list_component.js

```diff
@@ -0,0 +1,121 @@
+define([
+    '../applications_info',
+    'jsapi/v1/resources',
+    '../../dialogs'
+], function(applications_info, resources, dialogs) {
+    "use strict";
+    var available_applications_info =
+        applications_info.available_applications_info;
+
+    var Status = {
+        RUNNING: "running",
+        STARTING: "starting",
+        STOPPING: "stopping",
+        STOPPED: "stopped"
+    };
+
+    // Ember application model
+
+    const Application = Ember.Object.extend({
+        //
+    });
+
+    // Ember component for the application list
+
+    var ApplicationListComponent = Ember.Component.extend({
```

Comment:
The Ember component representing the list of applications. It contains all the data representing the applications.